src/main/java/uk/gov/di/ipv/cri/common/library/util/KMSSigner.java

```diff
@@ -42,25 +49,31 @@ public KMSSigner(String keyId, KmsClient kmsClient) {
     public Base64URL sign(JWSHeader header, byte[] signingInput) throws JOSEException {
         Objects.requireNonNull(signingInput, "Signing input must not be null");
 
-        byte[] signingInputHash;
+        SignResponse signResponse =
```

Comment:
Would it be better to be consistent with ipv-core and use the SignResult and AWSKMS class instead of the KmsClient?